components/romio/test/ome/io/nio/utests/PixelDataBitTest.java

```diff
@@ -28,46 +28,46 @@ public void setUp()
  
     public void testGetBits()
     {
-    	assertEquals(1.0, data.getPixelValue(0));  // Start byte 1
-    	assertEquals(1.0, data.getPixelValue(1));
-    	assertEquals(1.0, data.getPixelValue(2));
-    	assertEquals(1.0, data.getPixelValue(3));
-    	assertEquals(1.0, data.getPixelValue(4));
-    	assertEquals(1.0, data.getPixelValue(5));
-    	assertEquals(1.0, data.getPixelValue(6));
-    	assertEquals(1.0, data.getPixelValue(7));
-    	assertEquals(1.0, data.getPixelValue(8));  // Start byte 2 
-    	assertEquals(1.0, data.getPixelValue(9));
-    	assertEquals(0.0, data.getPixelValue(10));
-    	assertEquals(0.0, data.getPixelValue(11));
-    	assertEquals(0.0, data.getPixelValue(12));
-    	assertEquals(0.0, data.getPixelValue(13));
-    	assertEquals(0.0, data.getPixelValue(14));
-    	assertEquals(1.0, data.getPixelValue(15));
+        Assert.assertEquals(data.getPixelValue(0), 1.0);  // Start byte 1
+        Assert.assertEquals(data.getPixelValue(1), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(2), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(3), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(4), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(5), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(6), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(7), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(8), 1.0);  // Start byte 2 
+        Assert.assertEquals(1.0, data.getPixelValue(9), 1.0);
+        Assert.assertEquals(0.0, data.getPixelValue(10), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(11), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(12), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(13), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(14), 0.0);
+        Assert.assertEquals(1.0, data.getPixelValue(15), 1.0);
     }
     
     public void testSetAndGetBits()
     {
     	// 0
-    	assertEquals(1.0, data.getPixelValue(0));
+        Assert.assertEquals(data.getPixelValue(0), 1.0);
     	data.setPixelValue(0, 0.0);
-    	assertEquals(0.0, data.getPixelValue(0));
+    	Assert.assertEquals(data.getPixelValue(0), 0.0);
     	// 7
-    	assertEquals(1.0, data.getPixelValue(7));
+    	Assert.assertEquals(data.getPixelValue(7), 1.0);
     	data.setPixelValue(7, 0.0);
-    	assertEquals(0.0, data.getPixelValue(7));
+    	Assert.assertEquals(data.getPixelValue(7), 0.0);
     	// 8
-    	assertEquals(1.0, data.getPixelValue(8));
+    	Assert.assertEquals(data.getPixelValue(8), 1.0);
     	data.setPixelValue(8, 1.0);
-    	assertEquals(1.0, data.getPixelValue(8));
+    	Assert.assertEquals(data.getPixelValue(8), 1.0);
     	// 14
-    	assertEquals(0.0, data.getPixelValue(14));
+    	Assert.assertEquals(data.getPixelValue(14), 0.0);
     	data.setPixelValue(14, 1.0);
-    	assertEquals(1.0, data.getPixelValue(14));
+    	Assert.assertEquals(data.getPixelValue(14), 1.0);
```

Comment:
This will use `assertEquals(Object actual, Object expected)`, wonder if it would be worth changing to use the explicit `assertEquals(double actual, double expected, double delta)` for double comparison.